README.md

```diff
@@ -48,3 +48,12 @@ At first startup Eclipse IDE will checkout the code, perform a full build and co
 For further information, building and deployment instructions, please visit the [getting started page](https://wiki.eclipse.org/Kura/Getting_Started).
 
 To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+
+## Supported Development Platforms
+The Eclipse Installer based setup works for the main used platforms like Linux, Mac Os and Windows.
+
+
+## Known issues
+Currently, the emulator web ui is not properly working on Windows so, with your setup, you will be able to build and deploy you applications, but not be able to use the Eclipse IDE based Kura emulator.
+
+The full build of Kura is only supported for Linux and Mac Os based systems.
```

Comment:
I'd be more explicit saying that building the Kura installers require Linux or Mac OS

Reply:
Hi all, 
I wonder if we cannot use Eclipse IDE based on Kura emulator in windows then how could we build and deploy our package to Kura web UI e.g How can I build the demo heater example without Kura.
I have installed Eclipse Kura then import the developer workspace and it does not generate any error but, when I run Kura emulator it generates many errors. I have also tried to deploy the package that I created using Eclipse IDE but it fails to install in Kura. Can you guys help me with this?
1- How to install Kura in windows or how can I build the Kura example and deploy it?
Thank you

Regards,
Ahmad Arif

Reply:
Running Kura emulator on Windows is a long-standing issue (discussed in e.g. #637, #1023, #1204, etc.). Windows remains an unsupported production platform.
As far as development goes, Kura should build fine, including the examples. However, the emulator doesn't run properly.

The proper procedure that answers your question is therefore this:
* execute full Kura build, in cli
* if necessary, update heater demo in IDE, e.g. Eclipse
* build heater example in cli (`mvn clean install`) which will produce a .dp file
* run Kura on a supported platform and deploy the .dp file there

You can find more info [here](http://eclipse.github.io/kura).

Reply:
Hi markoer,
Thanks for the reply. I try to build heater example using mvn clean install but it gives errors, like 
"Failed to load p2 repository with ID  p2-repo-equinox3.1.11" from location. How to resolve this error?
Kind Regards,
Baba

Reply:
Let's just make one thing clear, first. You did a clone of git repository or you only have developer workspace from zip?
I suppose the former, since mvn actually complained about equinox repo. In this case, go to directory with build-all.sh (3 directories higher) and execute that. Once it's finished, you should already have the heater dp ready.
If the latter, then you're pretty much limited to Eclipse. You can usually get rid of the compilation errors by opening the kura-equinox_3.11.1.target file and Clicking on `Set as target platform` link.

Reply:
Let's take it from the start. I installed Eclipse IDE using Oopmh installer by following the instruction on Eclipse Kura documentation. When the installation has completed it was showing errors in target-definition/equinox 3.8.1/repository does not exist. After this, I import developer workspace and there is no error.
Now I edit demo heater example and run the emulator, It is showing these console logs.

18:39:12,649 [Camel (camel-4) thread #3 - timer://xmltopic] INFO  DataServiceImpl:531  - Storing message on topic :#account-name/#client-id/myapp/xmltopic, priority: 5
18:39:12,651 [Camel (camel-4) thread #2 - timer://heartbeat] INFO  DataServiceImpl:531  - Storing message on topic :#account-name/#client-id/myapp/topic, priority: 5
18:39:12,651 [Camel (camel-4) thread #3 - timer://xmltopic] INFO  DataServiceImpl:534  - Stored message on topic :#account-name/#client-id/myapp/xmltopic, priority: 5
18:39:12,652 [DataServiceImpl:Submit] INFO  DataServiceImpl:821  - DataPublisherService not connected
18:39:17,927 [MQTT Call: Deee] INFO  MESSAGE_FROM_CLOUD:159  - Exchange[ExchangePattern: InOnly, BodyType: org.eclipse.kura.message.KuraPayload, Body: org.eclipse.kura.message.KuraPayload@685c038b]
18:39:17,928 [MQTT Call: Deee] INFO  MESSAGE_SENT_TO_CLOUD:159  - Exchange[ExchangePattern: InOnly, BodyType: org.eclipse.kura.message.KuraPayload, Body: org.eclipse.kura.message.KuraPayload@685c038b]
18:39:17,930 [MQTT Call: Deee] INFO  lessThanTen:159  - Exchange[ExchangePattern: InOnly, BodyType: org.eclipse.kura.message.KuraPayload, Body: org.eclipse.kura.message.KuraPayload@685c038b]
18:39:18,649 [Camel (camel-4) thread #3 - timer://xmltopic] INFO  DataServiceImpl:531  - Storing message on topic :#account-name/#client-id/myapp/xmltopic, priority: 5
18:39:18,651 [Camel (camel-4) thread #3 - timer://xmltopic] INFO  DataServiceImpl:534  - Stored message on topic :#account-name/#client-id/myapp/xmltopic, priority: 5
18:39:18,651 [Camel (camel-4) thread #2 - timer://heartbeat] INFO  DataServiceImpl:531  - Storing message on topic :#account-name/#client-id/myapp/topic, priority: 5
18:39:18,653 [Camel (camel-4) thread #2 - timer://heartbeat] INFO  DataServiceImpl:534  - Stored message on topic :#account-name/#client-id/myapp/topic, priority: 5
18:39:18,920 [DataServiceImpl:Submit] INFO  MqttDataTransport:516  - Publishing message on topic: kapua-sys/Deee/myapp/topic with QoS: 0
18:39:18,926 [MQTT Call: Deee] INFO  CloudServiceImpl:436  - Message arrived on topic: kapua-sys/Deee/myapp/topic

When I see data at Eclipse kapua it updating the data (xmltopic) under myapp topic instead of updating the data under heater topic. This is really a teaser for me Kindly help.
Thank you

Reply:
Hi markoer,
The problem is solved thanks for the help.

Reply:
Hi markoer,
I want to read a sensor data from serial port of raspberry pi. I am following the serial example from Kura documentation but this is an error which keep coming up in eclipse console. 

org.eclipse.kura.KuraNotConnectedException: Not connected.
	at org.eclipse.kura.core.data.transport.mqtt.MqttDataTransport.unsubscribe(MqttDataTransport.java:474)
	at org.eclipse.kura.core.data.DataServiceImpl.unsubscribe(DataServiceImpl.java:525)
	at org.eclipse.kura.core.cloud.CloudClientImpl.unsubscribe(CloudClientImpl.java:200)
	at org.eclipse.kura.core.cloud.CloudClientImpl.unsubscribe(CloudClientImpl.java:193)
	at org.eclipse.kura.camel.cloud.KuraCloudConsumer.doStop(KuraCloudConsumer.java:57)
	at org.apache.camel.support.ServiceSupport.stop(ServiceSupport.java:102)
	at org.apache.camel.util.ServiceHelper.stopService(ServiceHelper.java:142)
	at org.apache.camel.impl.DefaultShutdownStrategy.shutdownNow(DefaultShutdownStrategy.java:381)
	at org.apache.camel.impl.DefaultShutdownStrategy$ShutdownTask.run(DefaultShutdownStrategy.java:575)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

Kindly help me how to resolve this?
Thanks